### PR TITLE
Hub 01 Surface Mercenary NPC

### PR DESCRIPTION
**Pull request by John-Candlebury**
#### Summary

```SUMMARY: Content "A new Hub 01 npc that give sidequests"```

#### Purpose of change

 The point of this NPC is to have someone to give out the more mundane fetch quests, so that the Intercom missions dont get bloated with boring stuff. 

#### Describe the solution

A fellow Hub 01 mercenaty spawns after completing the second intercom mission. They give missions. The first mission involves retrieving goldbars for the Hub´s mint.

#### Additional context

Later on they could probably be hired to help the player complete the main hub missions. But that will have to wait for another day.

### CODE REVIEW

File: data/json/npcs/robofac/NPC_ROBOFAC_MERC_1.json [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+    "id": "TALK_ROBOFAC_MERC_1_MAIN",
+    "type": "talk_topic",
+    "dynamic_line": [
+      "Rough out there isn't it?",
+      "Well?",
+      "Good to have some company, I guess.",
+      "Have something to say?",
+      "...",
+      "Didn't you want something?",
+      "..."
```
Comment (Night-Pryanik) [inline data/json/npcs/robofac/NPC_ROBOFAC_MERC_1.json]: Duplicate?

File: data/json/npcs/robofac/NPC_ROBOFAC_MERC_1.json [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+        "topic": "TALK_ROBOFAC_MERC_1_RANDOM_THOUGHTS"
+      },
+      { "text": "What do you know about our employers?", "topic": "TALK_ROBOFAC_MERC_1_ASK_HUB" },
+      { "text": "Well, I should get going.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "TALK_ROBOFAC_MERC_1_ASK_HUB",
+    "type": "talk_topic",
+    "dynamic_line": "Not much really.  They made me track and destroy some government documents a while back, of course I gave them a read first but there wasn't much.\n\n As far as I know, this place held some experimental computer network before everything went to shit.  Whats going on now is anybody's guess.",
```
Comment (Night-Pryanik) [inline data/json/npcs/robofac/NPC_ROBOFAC_MERC_1.json]:
```suggestion
    "dynamic_line": "Not much really.  They made me track and destroy some government documents a while back, of course I gave them a read first but there wasn't much.\n\n As far as I know, this place held some experimental computer network before everything went to shit.  What's going on now is anybody's guess.",
```

File: data/json/npcs/robofac/NPC_ROBOFAC_MERC_1.json [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+  {
+    "id": "TALK_ROBOFAC_MERC_1_GOING_ON",
+    "type": "talk_topic",
+    "dynamic_line": [ "Just came back from a job.  Trying to rest for a short while." ],
+    "responses": [ { "text": "...", "topic": "TALK_ROBOFAC_MERC_1_MAIN" } ]
+  },
+  {
+    "id": "TALK_ROBOFAC_MERC_1_COINS",
+    "type": "talk_topic",
+    "dynamic_line": "Have you noticed they pay us in tiny little gold coins, its bizarre I say.  One would think they have a gold mine in there, but no, they don't, so they make me cross the dammed world grabbing gold bars. ",
```
Comment (Night-Pryanik) [inline data/json/npcs/robofac/NPC_ROBOFAC_MERC_1.json]:
```suggestion
    "dynamic_line": "Have you noticed they pay us in tiny little gold coins, it's bizarre I say.  One would think they have a gold mine in there, but no, they don't, so they make me cross the damned world grabbing gold bars. ",
```

File: data/json/npcs/robofac/NPC_ROBOFAC_MERC_1.json [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+      "Have you ever noticed how... wait no, never mind.",
+      "I heard some strange dimensional resonance caused all this, do you think it has happened to other places?",
+      "You noticed this place has working WI-FI?  Well not the rest of the net works anyways."
+    ],
+    "responses": [ { "text": "...", "topic": "TALK_ROBOFAC_MERC_1_MAIN" } ]
+  },
+  {
+    "id": "TALK_ROBOFAC_MERC_1_GOLD_MISSION",
+    "type": "talk_topic",
+    "dynamic_line": "I can offer you the whole of my payment for every bar or equivalent you bring.  But keep silent about this, its supposed to be my job.",
```
Comment (Night-Pryanik) [inline data/json/npcs/robofac/NPC_ROBOFAC_MERC_1.json]:
```suggestion
    "dynamic_line": "I can offer you the whole of my payment for every bar or equivalent you bring.  But keep silent about this, it's supposed to be my job.",
```

File: data/json/npcs/robofac/NPC_ROBOFAC_MERC_1.json [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+    "responses": [
+      { "text": "We have a deal then", "effect": { "add_mission": "MISSION_ROBOFAC_MERC_1_GOLD" }, "topic": "TALK_DONE" },
```
Comment (Night-Pryanik) [inline data/json/npcs/robofac/NPC_ROBOFAC_MERC_1.json]:
```suggestion
      { "text": "We have a deal then.", "effect": { "add_mission": "MISSION_ROBOFAC_MERC_1_GOLD" }, "topic": "TALK_DONE" },
```

File: data/json/npcs/robofac/NPC_ROBOFAC_MERC_1.json [hunk 6]
```diff
@@ -0,0 +1,224 @@
...
+      { "text": "Actually, I don't think I have time for that, sorry.", "topic": "TALK_ROBOFAC_MERC_1_MAIN" }
+    ]
+  },
+  {
+    "id": "MISSION_ROBOFAC_MERC_1_GOLD",
+    "type": "mission_definition",
+    "name": "Retrieve gold bar",
```
Comment (Night-Pryanik) [inline data/json/npcs/robofac/NPC_ROBOFAC_MERC_1.json]: We're not looking specifically for a gold bar, right? Maybe `Retrieve chunks of gold`?

File: data/json/npcs/robofac/NPC_ROBOFAC_MERC_1.json (hunk 1 above)
Comment (John-Candlebury) [reply data/json/npcs/robofac/NPC_ROBOFAC_MERC_1.json]: To make "..." twice as likely.

**Pull request merged**